src/expressions/block-expr.md

```diff
@@ -117,6 +117,37 @@ loop {
 }
 ```
 
+## `const` blocks
+
+> **<sup>Syntax</sup>**\
+> _ConstBlockExpression_ :\
+> &nbsp;&nbsp; `const` _BlockExpression_
+
+A *const block* is a variant of a block expression which evaluates in the compile time instead of in the run time.
+
+A `const` block allows you to define a constant value without having to define a new `const` item, and thus is also sometimes called as inline `const` or anonymous `const`.
```

Comment:
Given the term defined is "const block", the quote ticks on subsequent usage is wrong. You may also consider giving it a name like "constant block" instead.

Same with const item. They are called "constant item" on their chapter.

Reply:
It seems that we have "constant item" but in the const eval chapter we use "const function", "const generic parameters" etc. I'll just keep it `const block` here.

The quote ticks are removed after term definition.